File: GTTS/INL.TrainingService.Models/GetTrainingEventBatch_Item.cs
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventBatch_Item : IGetTrainingEventBatch_Item
    {
        public int BatchNumber { get; set; }
        public List<GetTrainingEventBatchParticipants_Item> Participants { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-batch_item.ts


import { GetTrainingEventBatchParticipants_Item } from './get-training-event-batch-participants_item';
import { IGetTrainingEventBatch_Item } from './iget-training-event-batch_item';

export class GetTrainingEventBatch_Item implements IGetTrainingEventBatch_Item {
  
	public BatchNumber: number = 0;
	public Participants?: GetTrainingEventBatchParticipants_Item[];
  
}






